modules/latex.php

```diff
@@ -17,8 +17,137 @@
  * $latex [a, b]$              ->  [latex][a, b][/latex]
+ *
+ * In addition to supporting old markup, we also now support MathJax math delimiters:
+ * Inline: "\(math here\)"
+ * Block-level: "$$math here$$" and "\[math here\]"
+ * MathJax delimiters do not support colors and sizes as part of the expression.
+ */
+
+/**
+ * MathJax configuration.
+ *
+ * In-line MathJax configuration options give us more granularity than simply
+ * picking one of the built-in configurations. It also makes it filterable.
+ *
+ * @since 3.8
+ *
+ * @link http://docs.mathjax.org/en/latest/configuration.html#using-in-line-configuration-options
+ *
+ * @return array
+ */
+function latex_mathjax_config() {
+	$config = array(
+		// TeX-AMS_HTML.js allows math to be specified in TeX or LaTeX notation, with the AMSmath and AMSsymbols packages included, and produces output using the HTML-CSS output processor.
+		'config' => array( 'TeX-AMS_HTML.js' ),
+		'jax'    => array(
+			'input/TeX',
+			'output/HTML-CSS',
+			'output/CommonHTML',
+		),
+		'extensions' => array(
+			'tex2jax.js',
+			'CHTML-preview.js',
+			'Safe.js'
+		),
+		'TeX' => array(
+			'extensions' => array(
+				'AMSmath.js',
+				'AMSsymbols.js',
+			),
+			'Macros' => array(
+				// /i command sequence is supported by WP Latex but not MathJax.
+				'i' => "{\\imath}",
+				// /j command sequence is supported by WP Latex but not MathJax.
+				'j' => "{\\jmath}",
+			),
+		),
+		'tex2jax' => array(
+			'inlineMath' => array(
+				array( '\(', '\)' ),
+				array( '$latex ', '$' ),
+				array( '[latex]', '[/latex]' ),
+			),
+			'ignoreClass' => 'math_ignore',
+		),
+		// Specify a z-index for MathMenu so it appears properly when used inside modals.
+		'MathMenu' => array(
+			'styles' => array(
+				'.MathJax_Menu' => array('z-index' => 2001),
+			),
+		),
+	);
+
+	// Users who can edit posts or pages should see math parse error details.
+	if ( current_user_can( 'edit_posts' ) || current_user_can( 'edit_pages' ) ) {
+		$config['TeX']['noErrors']    = array( 'disabled' => true );
+		$config['TeX']['noUndefined'] = array( 'disabled' => true );
+	}
+
+	return $config;
+}
+
+/**
+ * Output in-line MathJax configuration.
+ *
+ * @since 3.8
+ *
+ * @link http://docs.mathjax.org/en/latest/configuration.html#using-in-line-configuration-options
+ * @link https://github.com/mathjax/MathJax-website/commit/df7f655623b1a6fc0bc637d9f4b47cb3461aa118
+ */
+function latex_mathjax_output() { ?>
+	<script type="text/x-mathjax-config">
+		MathJax.Hub.Register.StartupHook('MathMenu Ready', function () {
+			MathJax.Menu.BGSTYLE['z-index'] = 2000;
+		});
+		MathJax.Hub.Config(<?php echo wp_json_encode(
+			apply_filters( 'latex_mathjax_output', latex_mathjax_config() )
+		); ?>);
+	</script>
+<?php }
+
+/**
+ * Enqueue MathJax JavaScript.
+ *
+ * @since 3.8
  */
+function latex_enqueue_scripts() {
+	$mathjax_url = '//cdn.mathjax.org/mathjax/latest/MathJax.js';
```

Comment:
Ah, okay-- so your methodology seems fine for now-- we should probably set up the files on the wp.com CDN both for speed and ownership-- we don't want to be loading assets in the wild from third party servers we don't have any control over.  Should they go away or get hacked, all our users would be SOL